public/js/controllers/game.js

```diff
@@ -1,24 +1,64 @@
 angular.module('mean.system')
-  .controller('GameController', ['$scope', 'game', '$timeout', '$location', 'MakeAWishFactsService', '$dialog', function ($scope, game, $timeout, $location, MakeAWishFactsService, $dialog) {
+  .controller('GameController', ['$scope', 'game', '$timeout', '$location', 'MakeAWishFactsService', '$dialog', '$http', ($scope, game, $timeout, $location, MakeAWishFactsService, $dialog, $http) => {
     $scope.hasPickedCards = false;
     $scope.winningCardPicked = false;
     $scope.showTable = false;
     $scope.modalShown = false;
     $scope.game = game;
+    $scope.counter = 0;
     $scope.pickedCards = [];
-    var makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
+    let makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
     $scope.makeAWishFact = makeAWishFacts.pop();
+    $scope.test = 0;
+
+    $scope.hideModal = () => {
+      $('#searchInput').modal('hide');
+    };
+    $scope.endInvites = () => {
+      $('#end-invites').modal('show');
+    };
+    $scope.searchUser = (searchQuery) => {
+      if (searchQuery.trim().length) {
+        $http({
+          method: 'GET',
+          url: `/api/search/users/${searchQuery}`
+        })
+          .then((response) => {
+            const result = response.data.User;
+            $scope.searchResults = result;
+          });
+      }
+    };
+    $scope.sendInvites = (email, $index) => {
+      if ($scope.counter === 11) {
+        $scope.hideModal();
+        $scope.endInvites();
+        return;
+      }
+      $http({
+        method: 'POST',
+        url: '/api/users/invite',
+        data: {
+          email,
+          link: document.URL
+        }
+      }).then((response) => {
+        $scope.counter += 1;
+        console.log('current index is', $index);
+        console.log('counter is', $scope.counter);
+      });
+    };
 
-    $scope.pickCard = function(card) {
+    $scope.pickCard = (card) => {
       if (!$scope.hasPickedCards) {
         if ($scope.pickedCards.indexOf(card.id) < 0) {
           $scope.pickedCards.push(card.id);
           if (game.curQuestion.numAnswers === 1) {
             $scope.sendPickedCards();
             $scope.hasPickedCards = true;
           } else if (game.curQuestion.numAnswers === 2 &&
-            $scope.pickedCards.length === 2) {
-            //delay and send
+                        $scope.pickedCards.length === 2) {
```

Comment:
Either  move it to previous line
```js
else if (game.curQuestion.numAnswers === 2 && $scope.pickedCards.length === 2) {
  ...
}
```
Or break it as shown when the line is too long
```js
else if (
  game.curQuestion.numAnswers === 2 &&
  $scope.pickedCards.length === 2
) {
  ...
}